pages/platform/databases/postgresql_tuto_01_connect_strapi_to_managed_postgresql/guide.en-gb.md

```diff
@@ -0,0 +1,444 @@
+
+Click `Next`{.action} to continue.
+
+#### Step 3: Select a location
+
+Choose the geographical region of the data center in which your database will be hosted.
+
+![Choose region](images/postgresql-tuto-01-connect-strapi-to-managed-postgresql03.png){.thumbnail}
+
+Click `Next`{.action} to continue.
+
+#### Step 4: Configure database nodes
+
+You can choose the node model in this step. The initial and maximum numbers of nodes depend on the chosen solution in step 2.
+
+![Order nodes](images/postgresql-tuto-01-connect-strapi-to-managed-postgresql04.png){.thumbnail}
+
+Please visit the [capabilities page](https://docs.ovh.com/gb/en/publiccloud/databases/postgresql/capabilities/) for detailed information on the hardware resources and other properties of the MySQL database installation.
+
+Take note of the pricing information and click `Next`{.action} to continue.
+
+#### Step 5: Configure your options
+
+You can name your database in this step.
+
+![Configure options](images/postgresql-tuto-01-connect-strapi-to-managed-postgresql05.png){.thumbnail}
+
+Click `Next`{.action} to continue.
+
+#### Step 6: Summary and confirmation
+
+The last section will show a summary of your order as well as the API equivalent of creating this database instance with the [OVHcloud API](https://docs.ovh.com/gb/en/api/first-steps-with-ovh-api/).
+
+![Confirm order](images/postgresql-tuto-01-connect-strapi-to-managed-postgresql06.png){.thumbnail}
+
+Click `Create a database service`{.action} to continue.
+
+Within a few minutes your new database service will be deployed. Messages in the OVHcloud Control Panel will inform you when the database is ready to use.
+
+Your database is ready when cluster status is "Ready", node status is green, and number of users is set to "1 user".
+
+![Green statuses on Database Control Panel](images/postgresql-tuto-01-connect-strapi-to-managed-postgresql08.png){.thumbnail}
+
+### Authorise your IP address
+
+> [!warning]
+> For security reasons the default network configuration doesn't allow any incoming connections. 
+> To allow access from your application or development environment to your PostgreSQL database service, you must add your IP addresses to the service whitelist.
+
+#### Add your IPs to the DB authorised list
+
+Following the related documentation on how to [authorise the suitable IP addresses](https://docs.ovh.com/gb/en/publiccloud/databases/mongodb/managing-service/#configuring-authorised-ips), add your IPs to the authorised list.
+
+![Green statuses and authorized IPs setted on Database Control Panel](images/postgresql-tuto-01-connect-strapi-to-managed-postgresql10.png){.thumbnail}
+
+### Test connection to the PostgreSQL DataBase
+
+An easy and quick way to test the connection is using the Postgre Command Line Interface (CLI) [psql](https://docs.postgresql.fr/14/app-psql.html){.external}.
```

Comment:
"A quick and easy way to test the connection is to use the Postgre Command Line Interface (CLI)"

Reply:
Done